test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js

```diff
@@ -35,6 +35,9 @@ import xhrAuthenticatorEnrollEnableBiometricsSMS from '../../../playground/mocks
 
 import xhrSuccess from '../../../playground/mocks/data/idp/idx/success.json';
 
+const POLLING_INTERVAL = 4000;
+const RESEND_REMINDER_PROMPT_INTERVAL = 30000
```

Comment:
Is there any way we can shorten the polling interval in the mock file without causing issues in this test?

Reply:
Looks like it will cause issues in the tests; seems that there are tests that rely on the polling interval being 4 seconds.  Shortening the polling interval makes some tests poll too early, which will make them continue to another flow prematurely and causing assertions to fail.  I think the api mock variables would need to be refactored to do this.